Pinta.Core/Classes/Document.cs

```diff
@@ -40,13 +40,20 @@ namespace Pinta.Core
 	// somewhat arbitrary.  In general:
 	// Document - Data about the image itself
 	// Workspace - Data about Pinta's state for the image
-	public class Document
+	public class Document : ObservableObject
```

Comment:
OK, I'd guess it can probably be removed - if nothing anywhere is subscribing to the PropertyChanged even, then deriving from ObservableObject isn't necessary.